codejail_service/settings/base.py

```diff
@@ -26,218 +24,40 @@ def root(*path_fragments):
 # Application definition
 
 INSTALLED_APPS = (
-    'django.contrib.admin',
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
-    'release_util',
-)
-
-THIRD_PARTY_APPS = (
-    'corsheaders',
-    'csrf.apps.CsrfAppConfig',  # Enables frontend apps to retrieve CSRF tokens
-    'rest_framework',
-    'rest_framework_swagger',
-    'social_django',
-    'waffle',
-)
-
-PROJECT_APPS = (
     'codejail_service.apps.core',
     'codejail_service.apps.api',
 )
 
-INSTALLED_APPS += THIRD_PARTY_APPS
-INSTALLED_APPS += PROJECT_APPS
-
 MIDDLEWARE = (
-    # Resets RequestCache utility for added safety.
-    'edx_django_utils.cache.middleware.RequestCacheMiddleware',
-
     # Monitoring middleware should be immediately after RequestCacheMiddleware
-    'edx_django_utils.monitoring.DeploymentMonitoringMiddleware',  # python and django version
-    'edx_django_utils.monitoring.CookieMonitoringMiddleware',  # cookie names (compliance) and sizes
-    'edx_django_utils.monitoring.CachedCustomMonitoringMiddleware',  # support accumulate & increment
-    'edx_django_utils.monitoring.MonitoringMemoryMiddleware',  # memory usage
+    # TODO: re-enable once we have waffle available
+    # 'edx_django_utils.monitoring.DeploymentMonitoringMiddleware',  # python and django version
+    # 'edx_django_utils.monitoring.CookieMonitoringMiddleware',  # cookie names (compliance) and sizes
+    # 'edx_django_utils.monitoring.CachedCustomMonitoringMiddleware',  # support accumulate & increment
+    # 'edx_django_utils.monitoring.MonitoringMemoryMiddleware',  # memory usage
```

Comment:
Changes I made below:
1. I updated the TODO. I don't see how the original made sense, but I might be missing something.
2. I removed CookieMonitoringMiddleware.
3. I added proposed priorities for re-enabling into the comments. 
 
```
    # TODO: re-enable once waffle. which depends on a db, is optional. <INSERT TICKET LINK>

    # 'edx_django_utils.monitoring.DeploymentMonitoringMiddleware',  # python and django version (Re-enable Priority: P3) 
    # 'edx_django_utils.monitoring.CachedCustomMonitoringMiddleware',  # support accumulate & increment (Re-enable Priority: P5) 
    # 'edx_django_utils.monitoring.MonitoringMemoryMiddleware',  # memory usage (Re-enable Priority: P5)
```

Reply:
@timmc-edx: Please put 0 hours against resolving anything waffle related until we get a chance to speak on this topic. Thank you.